### DISCUSSION TITLE
MQTT Needs 5 char instead of 4 before

### DISCUSSION THREAD

**Discussion by spike318** [arendst/Tasmota#16348] [Q&A] [answered]
When i use the 12.1.1 firmware, i now need to put 5 char for my Mqtt pasword, My MQTT password is only 4 char. Is there an solution for this, because i have more than 30 modules working with this MQTT, i don't want to change them all.

Anny suggestions

**Comment by spike318** [comment]
![image](https://user-images.githubusercontent.com/74891969/186873664-1fd8eb08-6d13-43fb-bea7-7cbd4d06efda.png)

**Comment by barbudor** [comment, marked as answer]
Enter the password by command `MqttPassword` on the console instead of GUI
The length enforcement is only on the GUI

**Comment by spike318** [comment]
Thankz it works !
